_includes/post-macros.html

```diff
@@ -488,48 +437,78 @@ <h1 class="h4">{{ icon }} {{ header }}</h1>
 
 {%- set active_filters_total = active_filters|length + active_date_filters|length -%}
 
+{%- set filter_dict = {
+    'filters': filters,
+    'active_filters_total': active_filters_total
+} -%}
+
+<div class="expandable expandable__filters expandable__padded
+    {{ 'expandable__expanded' if active_filters_total > 0
+        and options.show_current_filters }}">
+    <button class="expandable_header expandable_target">
+        <span class="expandable_header-left expandable_label">
+            {{ options.expand_label }}
+        </span>
+        <span class="expandable_header-right expandable_link">
+            <span class="expandable_cue-open">
+                Show
+              <span class="cf-icon cf-icon-plus-round"></span>
+            </span>
+            <span class="expandable_cue-close">
+                Hide
+                <span class="cf-icon cf-icon-minus-round"></span>
+            </span>
+        </span>
+    </button>
+    <div class="expandable_content">
+       {{ filters_form(filters, query, doc_type, options) }}
+    </div>
+</div>
+
+{%- if options.show_current_filters -%}
+    {{ active_filters_helper(posts, filter_dict, options) }}
+{%- endif -%}
+
+{% endmacro %}
+
+
+{# A helper macro for filters()
+   ========================================================================== #}
+{% macro active_filters_helper(posts, filter_dict, options) %}
+
+{%- set filters = filter_dict.filters -%}
+{%- set active_filters_total = filter_dict.active_filters_total -%}
+{% from 'macros.html' import notification %}
+
 {%- if active_filters_total > 0 -%}
-    <ul class="filtered-by filtered-by__align-with-btn list__horizontal">
+    <div class="filtered-by">
         {%- if options.show_filters_label -%}
-        <li class="list_item filtered-by_header">
-        {%- if posts|list|length > 0 -%}
-            {{ posts.total }} filtered result{{ 's' if posts.total > 1 }} for
-        {%- else -%}
-            No results for
-        {%- endif -%}
-        </li>
-        {%- endif %}
-    {%- for filter in active_filters %}
-        <li class="list_item filtered-by_filter">
-            {{ 'Blog' if filter == 'post' else filter }}
-        {%- if not loop.last -%}
-            ,
-        {%- elif active_date_filters|length > 0 -%}
-            ,
+            {%- if posts|list|length > 0 -%}
+                {% set text =  posts.total | string
+                            + ' filtered result'
+                            + ('s' if posts.total > 1 else '') %}
+                {{ notification({
+                    'text': text,
+                    'type': 'success'
+                }) }}
+            {%- else -%}
+                {% macro _no_resuls_text (posts, filter_dict, options) %}
+                    <h3>Sorry, there were no results based
```

Comment:
I [added `short-desc` styles](https://github.com/cfpb/cfgov-refresh/pull/589/files#diff-4a1b733d46ab894d6953df312702af65R5183) in #589 if you want to just plan on using those.